src/ai.ts

```diff
@@ -4,13 +4,16 @@ export async function chatGPTRequest(prompt: string) {
     const configuration = new openai.Configuration({
         apiKey: process.env.OPENAI_API_KEY,
     });
-    
+
     const openaiClient = new openai.OpenAIApi(configuration);
 
     try {
         const gptResponse = await openaiClient.createChatCompletion({
             model: "gpt-3.5-turbo",
-            messages: [{role: "user", content: prompt}],
+            messages: [
+                { role: "system", content: "You are a programming expert. You can detect software development languages based on a set of files that represent a folder of source code." },
```

Comment:
Did you find that this yielded an improved result? I haven't found this to be necessary on my projects.

Reply:
Yes I did actually...on the C# one it failed to identify anything, adding this and it identified "C#, XML"